src/lib/subgraphHelpers.ts

```diff
@@ -81,7 +81,10 @@ export function pollSubgraphUpdating(): Observable<boolean> {
           // This only works if user is connected to web3, that's why we need the try
           let atCurrentBlock = true;
           try {
-            const curBlock = (await getCurrentBlock()).number;
+            const curBlock = (await getCurrentBlock())?.number;
+            if (curBlock === undefined) {
+              return; // try again next time
```

Comment:
maybe log something to the console..here?

Reply:
done